tensorzero_internal/src/gateway_util.rs

```diff
@@ -33,24 +34,28 @@ impl AppStateData {
     }
 }
 
-async fn setup_clickhouse(
+pub async fn setup_clickhouse(
     config: &Config<'static>,
     clickhouse_url: Option<String>,
+    run_migrations: bool,
 ) -> Result<ClickHouseConnectionInfo, Error> {
     if config.gateway.disable_observability {
         Ok(ClickHouseConnectionInfo::new_disabled())
     } else {
-        let clickhouse_url = clickhouse_url.ok_or_else(|| {
-            Error::new(ErrorDetails::AppState {
-                message: "Missing environment variable CLICKHOUSE_URL".to_string(),
```

Comment:
let's remove this run_migrations flag and just leave the functionality untested for now -- we'll address it in an upcoming PR around observability config options